### PR TITLE
Azure AD -> Entra

### PR DESCRIPTION
**Pull request by ckittel**
No functional changes

### CODE REVIEW

File: infra-as-code/bicep/main.bicep [hunk 1]
```diff
@@ -44,16 +44,16 @@ param frontendCloudInitAsBase64 string
 @secure()
 param adminPassword string
 
-@description('The Azure Active Directory group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
+@description('The Microsoft Entra group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```
Comment (josevarela) [inline infra-as-code/bicep/main.bicep:47]: I all previous instances the replacement name included ID at the end. Not sure if it was an oversight

```suggestion
@description('The Microsoft Entra ID group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

File: infra-as-code/bicep/vmss.bicep [hunk 2]
```diff
@@ -60,16 +60,16 @@ param vmssFrontendApplicationSecurityGroupName string
 @description('The name of the backend Application Security Group.')
 param vmssBackendApplicationSecurityGroupName string
 
-@description('The Azure Active Directory group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
+@description('The Microsoft Entra group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```
Comment (josevarela) [inline infra-as-code/bicep/vmss.bicep:63]: oversight?

```suggestion
@description('The Microsoft Entra ID group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

**Review by josevarela — CHANGES_REQUESTED** [review]
Small question/suggestion in-line. Feel free to overwrite if needed.

File: infra-as-code/bicep/main.bicep (hunk 1 above)
Comment (ckittel) [reply infra-as-code/bicep/main.bicep:47]: According to the product docs, it's "Microsoft Entra group" or "Microsoft Entra user"

So, if I have "ID" in another place... I need to fix it there. Can you point out where I made the other mistake by adding ID in front of "user" or "group" so I can fix it?

File: infra-as-code/bicep/vmss.bicep (hunk 2 above)
Comment (ckittel) [reply infra-as-code/bicep/vmss.bicep:63]: According to the product docs, it's "Microsoft Entra group" or "Microsoft Entra user"

So, if I have "ID" in another place... I need to fix it there.  Can you point out where I made the other mistake by adding ID in front of "user" or "group" so I can fix it?  These rules are hard to keep straight :)

File: infra-as-code/bicep/main.bicep (hunk 1 above)
Comment (josevarela) [reply infra-as-code/bicep/main.bicep:47]: You are good then. Just noticed the other instances are referencing the product in general, not a specific group or id

File: infra-as-code/bicep/vmss.bicep (hunk 2 above)
Comment (josevarela) [reply infra-as-code/bicep/vmss.bicep:63]: You are good then. Just noticed the other instances are referencing the product in general, not a specific group or id

**Review by josevarela — APPROVED** [review]
Looks good!

**Pull request merged**